cypress/integration/cbl/admin/tasks-manager.js

```diff
@@ -323,30 +323,31 @@ describe('CBL / Admin / Tasks Manager', () => {
             .within(() => {
 
                 // click cloned task selector
-                cy.extGet('slate-cbl-taskselector[name=ClonedTaskID]')
-                    .should('exist')
-                    .within(() => {
-
-                        cy.root()
-                            .get('.x-form-arrow-trigger')
-                            .click()
-
-                    });
+                cy.get('input[name="ClonedTaskID"]')
+                    .should('have.length', 1)
+                    .parents('.x-form-trigger-wrap')
+                        .should('have.length', 1)
+                        .within(() => {
+                            cy.get('.x-form-arrow-trigger')
+                                .click()
+                        });
 
                 // wait for options to load
                 cy.wait('@tasksData');
 
-                // verify shared task exists in picker dropdown
-                cy.extGet('slate-cbl-taskselector[name=ClonedTaskID]', { component: true })
-                    .then(selector => selector.getPicker().el.dom)
-                    .contains('.x-boundlist-item', 'A Shared Task')
-                    .should('exist');
-
-                // verify unshared task does not exist in picker dropdown
-                cy.extGet('slate-cbl-taskselector[name=ClonedTaskID]', { component: true })
-                    .then(selector => selector.getPicker().el.dom)
-                    .contains('.x-boundlist-item', 'An UnShared Task')
-                    .should('not.exist');
+                // get componentid in order to find foating options list outside of window scope
+                cy.get('input[name="ClonedTaskID"]')
+                    .then(($input) => {
+                        const componentID = $input.attr('data-componentid');
+
+                        cy.document().its('body').find(`#${componentID}-picker-listEl`)
+                            .contains('.x-boundlist-item', 'A Shared Task')
+                            .should('exist');
+
+                        cy.document().its('body').find(`#${componentID}-picker-listEl`)
+                            .contains('.x-boundlist-item', 'An UnShared Task')
+                            .should('not.exist');
+                    })
```

Comment:
eh I think it was better before, the dialog is modal and the selectors were sufficiently unique. There's just no point in them being inside the `within` so maybe just revert and move them below the `.within` so it's just sequenced after we wait for the window to be visible and there's no big window-wide `.within()` since we don't ned to do any multi-element DOM-based scoping

Reply:
I was questioning the necessity of doing it too.  I like the within aesthetically because it groups a bunch of assertions on elements in the form window, but then you have to bend over backwards with the dropdown because it's technically floating outside the window.  I put the original code back, but outside the within.